_pages/training/videos.md

```diff
@@ -10,39 +10,30 @@ redirect_from:
 
 <iframe title="Plain Language Summit 2020" width="560" height="315" src="https://www.youtube.com/embed/EsJh0GuGYDA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## YouTube playlists
+## Plain language playlists
 
-- [2019 PLAIN Summit](https://www.youtube.com/embed/cny1CIMuv3o)
-- [2018 PLAIN Summit](https://www.youtube.com/embed/W4gzfemW0Rc)
 - [PLAIN](https://www.youtube.com/playlist?list=PLd9b-GuOJ3nHMlmPFMw8cJxN_DW-odj0J)
 - [U.S. Citizenship and Immigration Services](https://www.youtube.com/playlist?list=PLADE80C67FDB39352)
 - [Purdue Online Writing Lab (OWL)](https://www.youtube.com/user/OWLPurdue/playlists)
 
 ## PLAIN Meetings
 
--  [June 2021 Meeting: Using Words to Build Better Digital Experiences](https://digital.gov/event/2021/06/09/plain-language-webinar-using-words-to-build-better-digital-experiences/)
+- [2019 PLAIN Summit](https://www.youtube.com/embed/cny1CIMuv3o)
+- [2018 PLAIN Summit](https://www.youtube.com/embed/W4gzfemW0Rc)
```

Comment:
Should this be a playlist instead of an embed? The video shows blown up.

Reply:
Yes, thank you. Will update.